### DISCUSSION TITLE
What features do you want to see?

### DISCUSSION THREAD

**Discussion by william-herring** [william-herring/bor#14] [Ideas]
# What does Bor need?

The great thing about Bor, is that it is open-source. Many other project/team management apps charge ridiculous prices for features that aren't even helpful. Please, comment what features you want to see, and react to others' ideas! I'm interested to see your ideas.

**Comment by GoZippy** [comment]
Hey I just found this on r/SelfHosted  Any work on this in the past 2 years?? What help do you need? Are you still working on this or similar? I really need to find better open alternative to Monday.com

**Comment by william-herring** [comment]
Hi. Unfortunately I have moved on from this project but feel free to clone
it and use it as a base for your own project :)
